prerendering.bs

```diff
@@ -528,11 +534,23 @@ Navigation redirects can also [=prerendering browsing context/activate=] [=prere
 
     1. Return.
 
-  1. If |browsingContext| is a [=prerendering browsing context=]  and <var ignore>currentURL</var>'s [=url/origin=] is not [=same origin=] with <var ignore>incumbentNavigationOrigin</var>, then:
+  1. If |browsingContext| is a [=prerendering browsing context=]  and <var ignore>currentURL</var>'s [=url/origin=] is not [=/same site=] with |incumbentNavigationOrigin|, then:
 
     1. Run the [=environment discarding steps=] for <var ignore>navigationParams</var>'s [=navigation params/reserved environment=].
 
     1. Return.
+
+  Additionally, append the following steps toward the end of the algorithm, after the steps which check <var ignore>locationURL</var>:
+
+  1. If |browsingContext| is a prerendering browsing context, and <var ignore>responseOrigin</var> is not [=same origin=] with |incumbentNavigationOrigin|, then:
+
+    1. Let |loadingModes| be the result of [=getting the supported loading modes=] for |response|.
+
+    1. If |loadingModes| does not [=list/contain=] \`<code><a for="Supports-Loading-Mode">credentialed-prerender</a></code>\`, then set |response| to a [=network error=].
+
+       <p class="note">This will cause the prerender to fail in the following steps of the algorithm.</p>
+
+       <p class="XXX">In the future we could possibly also allow the `uncredentialed-prerender` token to work here. However, since that is primarily intended for the cross site case, which isn't specified or implemented yet, we don't want to encourage its use in the wild yet, so implementations must reject the `uncredentialed-prerender` token for now.</p>
```

Comment:
Just to confirm: Is `class="XXX"` the correct identifier?

Reply:
I think it is OK (it basically means "TODO", and generates a red box like you can see in https://html.spec.whatwg.org/#navigating-across-documents:shared-declarative-refresh-steps ), but on second glance it might be nicer as a `class="note"`.